modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java

```diff
@@ -1476,6 +1476,15 @@ else if (_log.isDebugEnabled()) {
 							WorkflowConstants.STATUS_DRAFT)) {
 
 						modelInfo._jsonObject.put("showWorkflow", true);
+
+						if (_ctCollection.getStatus() ==
```

Comment:
Hi @ctang3 

I think you can forgo this entire if statement. The property `showWorkflowActions` is the check itself. If it's true the actions will be added and if false then they won't.

Reply:
Ah right, I completely overlooked that I had `_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT)` in the showWorkflowActions check, sorry! Pushed an update for it 😅